pom.xml

```diff
@@ -115,6 +116,7 @@
             <properties>
                 <cmake.min_osx_version>-DCMAKE_OSX_DEPLOYMENT_TARGET=10.9</cmake.min_osx_version>
                 <cmake.osx_arch>-DCMAKE_OSX_ARCHITECTURES=arm64</cmake.osx_arch>
+                <configured.classifier>mac-arm64</configured.classifier>
```

Comment:
All those arm classifiers are not same as the one in [here](https://github.com/trustin/os-maven-plugin#property-osdetectedarch), which is a recommended tool to auto check the os.arch. 

I am not sure about this, on one side, I think we can let user know the arch name we chose to be more clear about the arch (especially we have those armv6/7/8 linux build). One the other side, I think we can adapt what the auto tool find, like [here](https://github.com/awslabs/aws-crt-java/blob/main/src/main/java/software/amazon/awssdk/crt/CRT.java#L106)

Reply:
different arm versions of crt are not interchangeable, right? so we need need to extend the recommended tools list anyways.
im leaning more towards our own list, but clearly defining the name for it (ex. crt-platform.classifier) and specifying in the readme which ones we publish to maven

Reply:
the os-maven-plugin uses "osx" not "mac"